### ISSUE TITLE
Kinematicbody2d move function with wrong values on the dictionary

### ISSUE THREAD

**Issue by dmentation** [godotengine/godot#10107]
Windows 10 - Godot version:3.0

**Issue description:**
In kinematicbody2d the return of the method _move has some wrong values. In the code the mapping of the normal is wrong and the dictionarty will return the normal with the same value has the position.
On the line 951 of physics_body_2d.cpp there's the following code:

d["normal"] = col.collision;

but that's wrong because we need someting like this:

d["normal"] = col.**normal**;

I hope that I explained that well

**Comment by rubenjavier** [comment]
Asked to fix this some time ago https://github.com/godotengine/godot/issues/9397

also to please add collider_velocity to the Dictionary as follows
```
   d["normal"] = col.normal;
   d["collider_velocity"] = col.collider_vel;
```
Thanks in advance

**Comment by akien-mga** [comment]
Fixed by #10298.
